docs/dev/ovf2vtk.py

```diff
@@ -0,0 +1,42 @@
+import argparse
```

Comment:
Can we have at least one test for ovf2vtk? For example, conversion of an example file from ovf to vtk, and check the vtk file is created, and is likely to have the right data (for example check number of vectors in there). The very least this would detect if the script doesn't run.